recipes/bitsets/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "bitsets" %}
+{% set version = "0.7.16" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/b/bitsets/bitsets-0.7.16.zip 
```

Comment:
Could you use:

```
url: https://pypi.io/packages/source/b/bitsets/bitsets-{{ version }}.zip 
```

instead? I'm not sure if the hardcoded version will be handled properly by the regro-bot that automatically puts in a PR when the version is updated by the source.